README.md

```diff
@@ -85,26 +85,7 @@ Subdivision: [SPE] Sankt-Peterburg
 Coordinates: 59.88, 30.25
 ```
 
-# Building Debian package
-
-The most simple way is to run a make target
-
-```shell
-$ make debpackage
-```
-
-When packages are built, you can clean up the leftover with
-
-```shell
-$ dh clean
-```
-or
-```shell
-$ make debclean
-```
-
-
 ## License
 
-This project is licensed under the CC Attribution-ShareAlike 4.0 International -
-see the [LICENSE](LICENSE) file for details
+This project is licensed under the MIT license - see the [LICENSE.md](LICENSE.md)
```

Comment:
why `.md` file?

Reply:
Copy-pasted from NeoGo, works fine. Could've been simple LICENSE as well, but let it be a bit more fancy.

Reply:
well, never seen `.md` license (or never thought about it) but went through the first 10+ repos from the neofs-node's `go.mod`  and none of them use `.md`. also, it does not look like a markdown file in fact
but... if you think that version is acceptable too, i do not mind